NotifyMe.js

```diff
@@ -164,7 +164,7 @@ const NotifyMe = props => {
                         <Popover.Title as="h3">{heading}</Popover.Title>
                         <Popover.Content style={{ padding: '3px 3px' }}>
                             {showCount && <div>
-                                <Button variant="link" onClick={markAsRead}>
+                                <Button variant="link" onClick={props.markAsRead || markAsRead}>
```

Comment:
Can we rename the prop as `markAsReadFn` so that it sounds like a function?

Reply:
sure